sparsebit/quantization/modules/activations.py

```diff
@@ -202,3 +198,97 @@ def forward(self, x_in):
         x_in = self.input_quantizer(x_in)
         out = F.hardsigmoid(x_in, inplace=self.inplace)
         return out
+
+@register_qmodule(sources=[nn.MultiheadAttention])
```

Comment:
1. You should build a new file to describe the QMHSA. 
2. the module name -- transformer.py